source/blog/029-node-fibers-discontinued.md

```diff
@@ -0,0 +1,160 @@
+called "embedded Dart Sass". This involves running Dart Sass as a *subprocess*,
+rather than a library, and communicating with it using a special protocol. This
+provides several important improvements over the existing alternatives:
+
+* Unlike running `sass` from the command line, this will still work with plugins
+  like the webpack importer. In fact, we plan to match the existing JavaScript
+  API as closely as possible. This will probably run asynchronous plugins *even
+  faster* than synchronous ones.
+
+* Unlike the existing JS-compiled version, this will use the Dart VM. Due to the
+  more static nature of the Dart language, the Dart VM runs Sass substantially
+  faster than Node.js, which will provide about a 2x speed improvement for large
+  stylesheets.
+
+The Node.js host for Embedded Sass is still in active development, but there's
+[a beta release] avialable (with minimal features) if you want to kick the
+tires.
+
+[a beta release]: https://www.npmjs.com/package/sass-embedded
+
+### Worker Threads
+
+We've explored the possibility of running the pure-JS Dart Sass in a Node.js
+worker thread. Worker threads work a bit like fibers in that they make it
+possible for synchronous code to wait for asynchronous callbacks to run.
+Unfortunately, they're also *extremely* restrictive about what sorts of
+information can be passed across the thread boundary, which makes it much harder
+to use them to wrap a complex API like Sass's.
+
+At the moment, the Sass team is focused on Embedded Sass, so we don't have the
+spare bandwidth to dive into worker threads as an alternative. That said, we'd
+be happy to help a motivated user implement this. If you're interested, follow
+up on [the GitHub issue]!
+
+[the GitHub issue]: https://github.com/sass/dart-sass/issues/868
+
+### Reanimating `node-fibers`
+
+There's one other potential solution, although it would take true dedication to
+turn into reality. It would in principle be possible to add a new API to V8 that
+would *officially* support the hooks `node-fibers` needs to do its good work.
+This would allow the package to return gloriously to life and Sass to make
+`render()` fast on into the future.
+
+The Sass team has contacted both the V8 team and the owner of `node-fibers`, and
+both of them are amenable to this idea in principle. While neither one has the
+time to see it through to completion themselves, they've expressed willingness
+to help an engineer who's willing to give it a shot.
+
+This isn't a contribution for the faint of heart, though: it requires knowledge
+of C++, a willingness to learn at least the basics of the `node-fibers` codebase
+and V8's isolate APIs, and skills in both API design and human interaction to
+negotiate a stable API that will meet the needs of `node-fibers` *and* that the
+V8 team feels comfortable committing to maintain. But if you're interested,
+please don't hesitate to [reach out]!
+
+[reach out]: mailto:nweiz@google.com
```

Comment:
are you comfortable with your email here? I've experienced spambots picking up emails in the past so I'm wondering if you want to obfuscate it (some use JS, some just do `name at emailserver dot com`.

Reply:
Google's spam filters are usually pretty good. This email is publicly available in plaintext in a number of other places, so I'm not *too* worried about exposing it here.

Reply:
okie dokie! :)